src/Core/System/SystemConfig/MemoizedSystemConfigLoader.php

```diff
@@ -0,0 +1,68 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\SystemConfig;
+
+use Shopware\Core\System\SystemConfig\Event\SystemConfigChangedEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Contracts\Service\ResetInterface;
+
+class MemoizedSystemConfigLoader extends AbstractSystemConfigLoader implements EventSubscriberInterface, ResetInterface
```

Comment:
the loader should not be the subscriber, otherwise all decorations have to be an event subscriber too.

Symfony has some problems with decoration and event subscriber registration

Reply:
@dneustadt I think it is actually an issue regarding the tags and less the subscriber but you are correct, of course.

I have now changed it accordingly. Can you check if it is fine this way?

Reply:
Looks good. Please squash your commits afterwards.

Reply:
@dneustadt According to @shyim it should not be necessary to sqash them as your bot will do so anyway but done as well.